tutorials/analysis/vonBertalanffy/tutorial_vonBertalanffy.m

```diff
@@ -13,92 +13,15 @@
 % context, and (iv) thermodynamically constrained flux balance analysis. The theoretical 
 % basis for each of these methods is detailed within the cited papers.
 % 
-% *EQUIPMENT*
-% 
-% Ubuntu 16.04 LTS
-% 
-% Python 2.7
-% 
-% NumPy 1.11.1
-% 
-% ChemAxon MarvinBeans 16.9.5.0
-% 
-% OpenBabel 2.3
-% 
-% *EQUIPMENT SETUP*
-% 
-% The following commands are entered in a terminal window (bash or similar 
-% shell).
-%% Python 2 
-% OpenBabel only works with Python 2. Most distributions should already have 
-% this installed, but if this is not the case, the following lines will do it: 
-% 
-% sudo add-apt-repository ppa:fkrull/deadsnakes  
-% 
-% sudo apt-get update  
-% 
-% sudo apt-get install python2.7
-%% NumPy 
-% NumPy can be installed using the following commands: 
-% 
-% sudo apt-get install python-dev 
-% 
-% sudo apt-get install python-setuptools 
-% 
-% sudo wget http://downloads.sourceforge.net/project/numpy/NumPy/1.11.1/numpy-1.11.1.tar.gz  
-% 
-% sudo tar -xzvf numpy-1.11.1.tar.gz 
-% 
-% sudo cd numpy-1.11.1  
-% 
-% sudo python setup.py build -j 4 install 
-%% ChemAxon Calculator Plugin 
-% ChemAxon calculator plugin requires a license. Apply for an academic license 
-% at the following link: http://www.chemaxon.com/my-chemaxon/my-academic-license/ 
-% 
-% After your license has been made available, you can download from the “My 
-% Licenses” tab on the ChemAxon website. 
-% 
-% Download the license and place it under (replace USER by your actual user 
-% account): 
-% 
-% /home/USER/.chemaxon 
-% 
-% Download MarvinBeans for Linux, navigate to the directory where it was 
-% saved and make it executable (here, we downloaded version 16.9.5.0 - use the 
-% appropriate filename for your version):  
-% 
-% sudo chmod +x marvinbeans-16.9.5.0-linux_with_jre64.sh  
-% 
-% Execute the installer (again, use the same filename as above): 
-% 
-% sudo ./marvinbeans-16.9.5.0-linux_with_jre64.sh 
-% 
-% When asked for an installation directory, make it:  
-% 
-% /opt/ChemAxon/MarvinBeans 
-% 
-% This is important, since this is the path used by COBRA Toolbox. 
-% 
-% Finally, add the installation path to the PATH environment variable: 
-% 
-% PATH=$PATH:/opt/ChemAxon/MarvinBeans/bin 
-% 
-% sudo export PATH
-%% OpenBabel and Python bindings  
-% Install the OpenBabel and Python 2 bindings by entering the following: 
-% 
-% sudo apt-get install openbabel  
-% 
-% sudo apt-get install python-openbabel
+% **
```

Comment:
Also a Reference to the MD File (best including a link) here please.

Reply:
From what I know it is not yet decided how and where the .md file will be placed so I can write only a general comment saying that there is an installation documentation in a separate file for now and if it helps include a path.